addons/web/static/src/views/form/form_compiler.js

```diff
@@ -403,7 +403,7 @@ export class FormCompiler extends ViewCompiler {
     compileHeader(el, params) {
         const statusBar = createElement("div");
         statusBar.className =
-            "o_form_statusbar position-relative d-flex justify-content-between mb-md-2 pb-2 pb-md-0";
+            "o_form_statusbar position-relative d-flex justify-content-between mb-0 mb-md-2 px-3 pb-2 pb-md-0 px-md-0";
```

Comment:
Fixed the padding issue 

Concerning the action button dropdown, it will be handled in this [milk fix](https://www.odoo.com/web#id=3413078&cids=1&menu_id=4720&active_id=587&model=project.task&view_type=form)

@Brieuc-brd